UPGRADING.INTERNALS

```diff
@@ -158,8 +158,10 @@ PHP 8.4 INTERNALS UPGRADE NOTES
      once used (use with_pear variable name).
 
  c. Windows build system changes
-   - The configure options --with-oci8-11g, --with-oci8-12c, --with-oci8-19 have
-     been removed.
+   - The configure options --with-oci8-11g, --with-oci8-12c, --with-oci8-19,
+     --enable-apache2-2handler have been removed.
```

Comment:
It might not be used in the field anymore. Glad with the changes _or_ we can possibly make it an alias too then removing it for 9.x. wdyt

Reply:
Here I was thinking like this. That this --enable-apache2handler option is presumable not used anymore around. And this --enable-apache2-2handler is kind of like targeting specifically 2.2 version. With removing it, it makes the user aware that previous setup won't work anymore, if theoretically someone builds Apache 2.2. Now, there is mainly this --enable-apache2-4handler used, which I've added it like this so it is simpler to migrate and still possibly have (but this is not confirmed at all) Apache 2.6 at some point. Yes, I have no ideal solution here. Perhaps we can add it like an alias also... :thinking: